src/app.py

```diff
@@ -202,31 +220,38 @@ def load_file(filename: str) -> dict:
 
         # Rename the columns of the DataFrame
         df_repositories.columns = (
-            ["Repository", "Repository Type", "URL"] + selected_rules + ["Is Compliant", "Rules Broken"]
+            ["Repository", "Repository Type", "URL", "Created At"] + selected_rules + ["Is Compliant", "Rules Broken"]
         )
 
         st.subheader(":blue-background[Repository Compliance]")
 
+        # If there are no repositories within the selected time range, display an error message
+        if df_repositories.empty:
+            st.error("Nothing within that time range.")
+            st.stop()
+
         # Display the rules that are being checked
         st.write("Checking for the following rules:")
-
         col1, col2 = st.columns(2)
 
-        for i in range(0, len(selected_rules)):
-            if i % 2 == 0:
-                col1.write(f"- {selected_rules[i].replace('_', ' ').title()}")
-            else:
-                col2.write(f"- {selected_rules[i].replace('_', ' ').title()}")
-
-        with st.expander("See Rule Descriptions"):
-            st.subheader("Rule Descriptions")
+        with col1:
```

Comment:
Im not sure if having the 2 expanders side by side is UI friendly here. It might be a better idea to always have them stacked (i.e if the right hand side expander is expanded, there will be a large gap in the left portion of the screen). I'll double check the UI on Friday.

Putting the selected rules in an expander is a good change though 👍

Reply:
I could put them in one expander that says 'About Rules'. It could then display the chosen rules and all the rule definitions. An expander 100% width, with 2 columns within the expander?

Reply:
I've had a look, I would probably have 2 expanders stacked on top of each other. Having the two side by side creates a load of dead space if you expand the right one.

Just get rid of the columns and it'll be good

Reply:
I'd also move the "checking for the following rules:" into the selected rules expander

Reply:
[Stacked the expanders](https://github.com/ONS-Innovation/github-policy-dashboard/pull/16/commits/ec2ac94393750bc5fa40d5fd5d197ea9965f73a2) and [moved text into expander](https://github.com/ONS-Innovation/github-policy-dashboard/pull/16/commits/5ec30519cd86995a3e853ee4395cdf31db791fc7).